src/main/features/response/helpers/taskListBuilder.ts

```diff
@@ -41,11 +41,6 @@ export class TaskListBuilder {
     const tasks: TaskListItem[] = []
     const externalId: string = claim.externalId
 
-    if (!ClaimFeatureToggles.isFeatureEnabledOnClaim(claim)
```

Comment:
Should the test for ` (draft.isResponsePartiallyAdmitted() || draft.isResponseFullyAdmitted())` remain?

Reply:
No because this condition applies only if there is no admissions feature in claim and if you see response type PA of FA. After making admission as a default feature we don't

Reply:
No because this condition applies only if there is no admissions feature in claim and if you see response type PA of FA. After making admission as a default feature we don't need to check response type as it is allowed always and no need to delete from draft